output file:
processed_torchgeoposition_embedding137.json
function:
position_embedding
Error Cases:

Pass or Failed: 1

Related Failed Test Cases:
set()

All Test Cases On Generated code:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/torchgeo/torchgeo/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/torchgeo/torchgeo
configfile: pyproject.toml
plugins: hydra-core-1.3.2
collecting ... collected 16 items / 2 deselected / 14 selected

../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths0] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths1] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths2] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths3] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths4] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths5] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFASmall16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa_weights[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_transforms[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa_weights[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_transforms[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFAHuge16::test_dofa PASSED

====================== 14 passed, 2 deselected in 18.63s =======================


Final Test Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/torchgeo/torchgeo/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/torchgeo/torchgeo
configfile: pyproject.toml
plugins: hydra-core-1.3.2
collecting ... collected 16 items / 2 deselected / 14 selected

../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths0] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths1] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths2] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths3] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths4] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths5] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFASmall16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa_weights[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_transforms[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa_weights[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_transforms[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFAHuge16::test_dofa PASSED

====================== 14 passed, 2 deselected in 17.44s =======================


Initial Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/torchgeo/torchgeo/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/torchgeo/torchgeo
configfile: pyproject.toml
plugins: hydra-core-1.3.2
collecting ... collected 16 items / 2 deselected / 14 selected

../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths0] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths1] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths2] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths3] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths4] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFA::test_dofa[wavelengths5] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFASmall16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_dofa_weights[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFABase16::test_transforms[DOFABase16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_dofa_weights[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFALarge16::test_transforms[DOFALarge16_Weights.DOFA_MAE] PASSED
../publishablew/torchgeo/torchgeo/tests/models/test_dofa.py::TestDOFAHuge16::test_dofa PASSED

====================== 14 passed, 2 deselected in 18.30s =======================
